packages/universal-router-sdk/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@pancakeswap/universal-router-sdk",
+  "version": "1.0.0",
+  "description": "🛠 An SDK for building applications on top of Pancakeswap.",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "module": "dist/index.mjs",
+  "files": [
+    "dist"
+  ],
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pancakeswap/pancake-frontend.git",
+    "directory": "packages/universal-router-sdk"
+  },
+  "keywords": [
+    "pancakeswap"
+  ],
+  "scripts": {
+    "build": "tsup",
+    "dev": "tsup --watch",
+    "coverage": "vitest run --coverage",
+    "test": "vitest --run",
+    "prepublishOnly": "pnpm run build",
+    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist",
+    "format:check": "prettier --check --loglevel error \"src/**/*.{js,jsx,ts,tsx,json}\"",
+    "format:write": "prettier --write \"src/**/*.{js,jsx,ts,tsx,json}\""
+  },
+  "author": "@Pancakeswap",
+  "devDependencies": {
+    "@pancakeswap/tokens": "workspace:*",
+    "@types/node": "^15.12.2",
+    "tslib": "^2.3.0",
+    "tsup": "^6.7.0",
+    "typescript": "^5.1.3",
+    "vitest": "^0.30.1"
+  },
+  "dependencies": {
+    "@pancakeswap/chains": "workspace:*",
+    "@pancakeswap/permit2-sdk": "workspace:*",
+    "@pancakeswap/sdk": "workspace:*",
+    "@pancakeswap/smart-router": "workspace:*",
+    "@pancakeswap/swap-sdk-core": "workspace:*",
+    "@pancakeswap/v3-sdk": "workspace:*",
+    "abitype": "^0.9.8",
+    "tiny-invariant": "^1.1.0",
+    "viem": "^1.10.9"
```

Comment:
make sure we're using the same version as the other packages

Reply:
yep bumped viem